docs/kb/semgrep-ci/bitbucket-triggering-events-to-jenkins.md

```diff
@@ -0,0 +1,38 @@
+---
+tags:
+  - Bitbucket
+  - Jenkins
+  - webhooks
+description: How to trigger events from the Bitbucket server to a Jenkins instance
+---
+# Triggering events from Bitbucket to Jenkins
+
+## Motivation
+
+A common scenario for many organizations is having:
+* Bitbucket (Cloud or Server) as Source Code Management system to manage source code repositories.
+* Jenkins as CI/CD platform to run pipelines.
+So, it is crucial to have some kind of links between the two platforms.
+
+## Defining a Bitbucket webhook 
+One way to connect Bitbucket (Cloud or Server) to a Jenkins instance is defining a Bitbucket webhook.
+It will allow us to trigger builds on Bitbucket's push and pull requests events.
+
+There are two requirements to comply:
+* Installing the plugin ["Bitbucket Push and Pull Request"](https://plugins.jenkins.io/bitbucket-push-and-pull-request/) on the Jenkins server.
+* Having network visibility between the two platforms.
```

Comment:
```suggestion
* Network visibility between the two platforms.
:::
```
What does network visibility mean? Are there permissions, IP addresses, etc. that need to be configured?